<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DownloadingInfoWidget</class>
 <widget class="QWidget" name="DownloadingInfoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DownloadingInfoWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="3" column="0">
    <widget class="QLabel" name="labelSpeedIdentifier">
     <property name="text">
      <string>Speed:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="3">
    <widget class="QLabel" name="labelTotalSize">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="3">
    <widget class="QLabel" name="labelSpeed">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="3">
    <widget class="QLabel" name="labelFileName">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="7" column="1" colspan="3">
    <widget class="QLabel" name="labelFolderPath">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="labelFileNameIdentifier">
     <property name="text">
      <string>FileName:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="labelDownloadedIdentifier">
     <property name="text">
      <string>Downloaded:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="totalSizeLabel">
     <property name="text">
      <string>TotalSize:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelURIIdentifier">
     <property name="text">
      <string>URI:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="3">
    <widget class="QLabel" name="labelRemainingTime">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="labelFolderPathIdentifier">
     <property name="text">
      <string>FoloderPath:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="QLabel" name="labelURI">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="3">
    <widget class="QLabel" name="labelDownloaded">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="labelRemainingTimeIdentifier">
     <property name="text">
      <string>RemainingTime</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="btnClose">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
         <iconset theme=":/icon/close.svg"/>
     </property>
     <property name="_vanillaStyle_Patch" stdset="0">
      <string>CleanButtonPatch</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
